draft-ietf-mls-protocol.md

```diff
@@ -954,9 +1003,11 @@ following primitives to be used in group key computations:
 * A hash algorithm
 * A signature algorithm
 
-MLS uses draft-08 of HPKE {{I-D.irtf-cfrg-hpke}} for public-key encryption.
-The `DeriveKeyPair` function associated to the KEM for the ciphersuite maps
-octet strings to HPKE key pairs.
+MLS uses draft-08 of HPKE {{I-D.irtf-cfrg-hpke}} for public-key encryption.  The
```

Comment:
Thi reintroduces HPKE draft-08. I guess #512 should be merged after this PR.

Reply:
I merged #512 so this got fixed as a conflict resolution.